### DISCUSSION TITLE
timer problem on pug_ready_type "0"

### DISCUSSION THREAD

**Discussion by itsmahdiso** [SmileYzn/PugMod-AMXX#48] [General]
When I set the game start mode to timerr, after the presence of the players on the server, a timer appears at the top of the screen that indicates when the game starts.

In the settings, we set the game to start after 60 seconds, but the timer shows (30:59)
The game starts exactly after one minute but the timer shows the wrong number (30 minutes more)
How can this problem be fixed?

**Comment by SmileYzn** [comment]
Did you downloaded from releases section?
Maybe it is a problem with format_time from amxmodx.

What is your timezone bro?

**Reply by itsmahdiso** [reply]
- yes i download last version from releases
- tehran

how can i fix that?

**Comment by SmileYzn** [comment]
Try to change 

				new szTime[16];
				format_time(szTime,charsmax(szTime),"%M:%S",iRemainTime);

to

				new szTime[32];
				format_time(szTime,charsmax(szTime),"%X",iRemainTime);

And see if it works

Ps. Post problems at Issues sections, that is more easy to manage the responses :+1:

**Reply by itsmahdiso** [reply]
the problem was not solved
not timer started with 03:30:59
maybe this is from tehran timezone!
our timezone is +3:30
how can we fix it?

**Reply by SmileYzn** [reply]
Try with "%EX" format. Or just try to put -3 at get_systime.

But if time is correctly calculated, do not make sense the format_time display based on timezone offset..

**Reply by itsmahdiso** [reply]
Could not be resolved

**Comment by SmileYzn** [comment]
Try to put 0 at get_systime or other values, to see what's happen.
